src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -658,6 +732,13 @@ static int dhcp6_lease_parse_message(
 
                         irt = unaligned_read_be32(optval) * USEC_PER_SEC;
                         break;
+
+                case SD_DHCP6_OPTION_VENDOR_OPTS:
+                        r = dhcp6_lease_add_vendor_sub_option(lease, optval, optlen);
```

Comment:
Why the function is named `sub` ? Please use the consistent name with the enum.

Reply:
removed sub from function name